docs/tutorials/tutorial0.md

```diff
@@ -80,6 +80,13 @@ mkdir ~/janis
 cd ~/janis
 ```
 
+Let's also create a directory to store the files for the tutorials.
+
+```bash
+mkdir janis-tutorials
+cd janis-tutorials
```

Comment:
Creating `janis-tutorials` in tutorial 0. Assuming users will follow the tutorials in order (added a note to each about it, if not already there).